src/Microsoft.Framework.PackageManager/Bundle/BundleOperations.cs

```diff
@@ -17,15 +18,40 @@ public void Delete(string folderPath)
             FileOperationUtils.DeleteFolder(folderPath);
         }
 
-        public void Copy(string sourcePath, string targetPath)
+        public void Copy(IEnumerable<string> sourceFiles, string sourceDirectory, string targetDirectory)
         {
```

Comment:
This is a public method on a public class - doesn't it need some argument Null checking?